ra_aid/prompts.py

```
@@ -431,7 +458,7 @@
     Do not suggest fixes or improvements.
     Do not mention what should be done.
     Do not discuss how the code could be better structured.
-    Do not provide advice or commentary on the project's future.
+    Do not provide advice or commentary on the project’s future.
```

Ruff (0.8.2)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)